valhalla/baldr/pathlocation.h

```diff
@@ -78,6 +78,13 @@ struct PathLocation : public Location {
    */
   bool shares_edges(const PathLocation& other) const;
 
+  /**
+   * Check if path edge has high reachability for this location.
+   * @param edge PathEdge that should be checked
+   * @return     true if the edge is high reachable
+   */
+  bool is_high_reachable(const PathEdge& edge) const;
```

Comment:
seems this method is actually unused. we instead use the free function inside thor route_action

Reply:
will remove